RecoLocalTracker/SiPixelRecHits/python/SiPixelRecHits_cfi.py

```diff
@@ -7,9 +7,6 @@
     VerboseLevel = cms.untracked.int32(0),
 )
 
-from RecoLocalTracker.SiPixelRecHits.siPixelRecHitHeterogeneous_cfi import siPixelRecHitHeterogeneous as _siPixelRecHitHeterogeneous
-gpu.toReplaceWith(siPixelRecHits, _siPixelRecHitHeterogeneous)
-
```

Comment:
This customization should be kept, otherwise the heterogeneous rechits are not run.

Reply:
they are run as explicitely introduced in RecoLocalTracker_cff.py
```
_pixeltrackerlocalreco_gpu.replace(siPixelRecHitsPreSplitting,siPixelRecHitHeterogeneous+siPixelRecHitsPreSplitting)
```

I suppose I am missing something

Reply:
No, sorry, I got confused.